src/Aquarius.Client/TimeSeries/Client/ServiceModels/Provisioning.cs

```diff
@@ -126,6 +126,21 @@ public class PublicKey
         public string Xml { get; set; }
     }
 
+    public enum AuditEventType
+    {
+        Create,
+        Update,
+        Delete,
+    }
+
+    public enum AuditType
+    {
+        All,
```

Comment:
Does the All AuditType still belong in this enum?

Reply:
Good check. I see what you mean, we did opt to not use it on the FE side. It's totally fine here.